examples/documents/query-paginated.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021, Nitric Technologies Pty Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [START import]
+import { documents } from '@nitric/sdk';
+// [END import]
+
+export async function queryPaginatedDocument() {
+  // [START snippet]
+  const docs = documents();
+
+  const query = docs
+    .collection('Customers')
+    .query()
+    .where('active', '==', true)
+    .limit(100);
+
+  // Fetch first page
+  let results = await query.fetch();
+
+  // Fetch next page
+  results = await query.pagingFrom(results.pagingToken).fetch();
```

Comment:
Suggest wrapping this in an `if` statement to make it clear that fetching additional pages should only be done if there are more available.

```typescript
if (results.pagingToken) {
  results = await query.pagingFrom(results.pagingToken).fetch();
}
```